### ISSUE TITLE
renaming properties is not detected as (breaking) change

### ISSUE THREAD

**Issue by dloibl** [pb33f/openapi-changes#80]
Hi,

first thanks for this awesome project! When using openapi-changes I noticed that renaming properties are not detected at changes. Removal or changing types is detected correctly.

Example to reproduce:

`openapi-changes summary original.yaml changed.yaml`

original.yaml
```yaml
openapi: 3.1.0
paths:
  /api/header:
    get:
      responses:
        '200':
          content:
            application/json:
              schema:
                type: object
                properties:
                  name:
                    type: string


```

changed.yaml
```yaml
openapi: 3.1.0
paths:
  /api/header:
    get:
      responses:
        '200':
          content:
            application/json:
              schema:
                type: object
                properties:
                  text: # breaking change!
                    type: string

```
output:  SUCCESS  No changes detected

should be:   ERROR   breaking changes discovered

**Comment by daveshanley** [comment]
Thank you for your kind words!

This is strange, I will look into this for you.

**Comment by daveshanley** [comment]
This has been resolved in `v0.0.52`

Here is the output using these example specs.

![Screenshot 2024-01-05 at 8 54 32 AM](https://github.com/pb33f/openapi-changes/assets/187345/043d4226-8f83-4d32-ae48-eab9aacff379)
